docs/00a-preamble/00-introduction.adoc

```diff
@@ -1,5 +1,5 @@
 // tag::DE[]
-== Einführung: Allgemeines zum iSAQB Advanced Level
+== Einleitung: Allgemeines zum iSAQB-Advanced-Level
```

Comment:
Das sollte "Einführung" bleiben, damit es bei allen Modulen gleich ist. 

Das mit den Bindestrichen ist ein guter Hinweis, das könnte ich dann in das Template und alle anderen Repos übernehmen.